bokeh/settings.py

```diff
@@ -326,6 +348,12 @@ def nodejs_path(self, default=None):
 #:   to an affirmative value will result in more friendly simple numeric IDs
 #:   counting up from 1000.
 #:
+#: ``BOKEH_USE_BINARY_ARRAYS`` --- Whether to encode data arrays before sending them.
+#:
+#:   Accepted values are ``yes``/``no``, ``true``/``false`` or ``0``/``1``.
+#:   Can have a huge effect on performance because encoding and decoding
+#:   arrays as raw strings can be very slow.
```

Comment:
also, if binary allows Nan/Infinity to work, that's a semantic difference right, which isn't ideal to have a config option affecting...